anaconda_packaging_utils/api/jira_api.py

```diff
@@ -0,0 +1,76 @@
+"""
+File:           jira_api.py
+Description:    Wrapper library that provides tools for using the Python JIRA API. The primary purpose of this wrapper
+                is to simplify and standardize the authentication process.
+
+                JIRA API links:
+                  - Source: https://github.com/pycontribs/jira
+                  - Docs: https://jira.readthedocs.io/index.html#
+"""
+import logging
+from typing import Final
+
+from jira.client import JIRA
+
+from anaconda_packaging_utils.storage.config_data import ConfigData
+
+# Logging object for this module
+log = logging.getLogger(__name__)
+
+# Where our JIRA boards are hosted
+_JIRA_HOST_URL: Final[str] = "https://anaconda.atlassian.net/"
```

Comment:
even though it starts with `_` it is global var, consider making it as static JiraApi class variable ?

Reply:
The linter should catch external access but I'll go ahead and move it. I think in python they call it a "class member variable" which is the closest we have to a C++ static.